includes/Admin/ReviewNotice.php

```diff
@@ -0,0 +1,152 @@
+                            /* translators: %d is replaced with the number of campaigns */
+                            printf( __( 'You sent %d campaigns usign weMail successfully.', 'wemail' ), $this->campaign_count );
+                        }
+                        ?>
+                    </h3>
+                    <p>
+                        <?php
+                        echo __(
+                            'Don’t worry, we are not asking you to buy the pro plan. But if you give us a great review, if you will. We put a lot of
+                            hard work to develop it and make it better every day. It would motivate the team a lot! Do you think it’s fair?', 'wemail'
+                        );
+                        ?>
+                    </p>
+                    <div class="wemail-reivew-notice-connect-button" style="margin-bottom: 10px: margin top: 10px">
+                        <form action='' method='post'>
+                            <button type="submit" class="button review_reposnse_yes" id="review_reposnse_yes" name="review_reposnse_yes">
+                                <?php echo __( 'Yes, Absolutely', 'wemail' ); ?>
+                            </button>
+                            <button type="submit" class="button response-button" id="review_reposnse_later" name="review_response_later">
+                                <?php echo __( 'Ask me later', 'wemail' ); ?>
+                            </button>
+                            <button type="submit" class="response-button" id="review_reposnse_no" name="review_response_no">
+                                <u> <?php echo __( 'No, I\'m just taking, not giving', 'wemail' ); ?> </u>
+                            </button>
+                            <?php wp_nonce_field( '_review_nonce', 'review_nonce' ); ?>
+                        </form>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <?php
+    }
+
+    /**
+     * Show notice to connect site in weMail
+     *
+     * @return void
+     */
+    public function connect_review_notice() {
+        if ( isset( $_GET['dismiss_wemail_review_notice'] ) && (int) $_GET['dismiss_wemail_review_notice'] === 1 ) {
+            if ( ! isset( $_GET['review_nonce'] ) ) {
+                wp_die( __( 'Nonce not Found', 'wemail' ) );
+            }
+
+            if ( ! wp_verify_nonce( sanitize_text_field( wp_unslash( $_GET['review_nonce'] ) ), '_review_nonce' ) ) {
+                wp_die( __( 'Security check failed', 'wemail' ) );
+            }
+
+            update_option( 'wemail_review_notice', 1 );
+		}
+
+        $installed_time = get_option( 'wemail_installed_time' );
+
+        $this->handle_review_response();
+
+        $this->check_campaign();
+
+		if ( (int) get_option( 'wemail_review_notice' ) !== 1 ) {
+			$this->time_based_review = ( time() - $installed_time ) / 86400 > 7;
+			$this->day_count = (int) ( time() - $installed_time ) / 86400;
```

Comment:
If any user installed weMail very long time ago, then he could be seen very long days. like: `898 days`. In that case, we could display it like: `more than 1 month` or something human-readable.